src/main/scala/xiangshan/mem/lsqueue/LoadMisalignBuffer.scala

```diff
@@ -225,36 +213,72 @@ class LoadMisalignBuffer(implicit p: Parameters) extends XSModule
           // need replay or still has unsent requests
           bufferState := s_req
         } .otherwise {
-          // merge the split load results
-          bufferState := s_comb
+          when(!req.isvec) {
+            // merge the split load results
+            bufferState := s_comb_wakeup_rep
```

Comment:
Here write `bufferState=s_comb_wakeup_rep` once under this otherwise condition, which may be clearer.

Reply:
ok, you're right.